lib/AvatarPresence/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import Tooltip from '../Tooltip';
+
+const tooltipOnHover = (data, isAssigned) => (
+  <Tooltip className="avatar__tooltip" id={data.email}>
+    <p className="avatar__tooltip-name">{data.name}</p>
+    <p className="avatar__tooltip-email">{data.email}</p>
+    { isAssigned &&
+      <span className="avatar__pillbox">Assigned</span>
+    }
+  </Tooltip>
+);
+
+const AvatarPresence = (props) => {
```

Comment:
Adding a little more to this I forgot to mention your comment;

> I expect some changes to be requested regarding the existing solo Avatar component, which now sits in isolation. I don't think there's a need to keep it around, as it's easily replaceable with this new one. Will likely rename the AvatarPresence to just Avatar... thoughts?

In my opinion there's still a need for the original Avatar component. AvatarPresence as it currently stands does 2 things;

- renders an avatar
- renders a tooltip

What if I wanted an Avatar without a tooltip? We could add extra props but I think its much cleaner to have 2 separate components one responsible for rendering an Avatar and handling all the custom options it supports (colours etc...) and then another component which uses the Avatar & Tooltip component. 

Again look at how `ButtonWithTooltip` works, the component name is super clear and functionality its just using component composition which is again easy to understand.